README.md

```diff
@@ -1 +1,204 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install
+ ```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+    cd hello_microverse_project
+    git clone https://github.com/peter-mwovi/hello_microverse_project.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello_microverse_project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+    npm run build (it will create a production ready file)
+    npm start (it will run the server on your machine)
+```
+
+### Run 
+
+I have added linters to ensure proper practice in the project.
+
+To run the linter tests on ``` index.html ```,  run the following command:
+
+```sh
+    npx hint .
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+    https://vercel.com/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Peter MWovi**
+
+- GitHub: [@githubhandle](https://github.com/peter-mwovi/)
+- Twitter: [@twitterhandle](https://twitter.com/mwovi_peter)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/peter-mwovi-57141a179/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Landing_page]**
+- [ ] **[Admin_dashboard]**
+- [ ] **[nBackend]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please follow me on my social media handles then go ahead and contribute.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing me with the necessary knowledge to work on such a project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ  -->
+
+## ❓ FAQ <a name="faq"></a>
+
+
+- **What is Linters?**
+
+  - Linters is a tool to analyse Source code so as to flag errors.
+
+- **Do I need knowledge in javascript to contribute?**
+
+  - basic knowledge in javascript is required. However feel free to play around with the source code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] You have done exceptionally great, You just need to update a broken link before moving to the next milestone. Currently, the MIT license link in your readme file leads to a 404 error page. A broken link in a readme file makes it look unprofessional, I'll suggest you create a `MIT.md` file and add the license content, You can [use this ](https://github.com/V-Blaze/gvash-planes/blob/dev/LICENSE)and then customize it for your project. :+1: :smile: 

- - 
![Screenshot from 2023-05-30 13-47-04](https://github.com/fatima-najafi/Hello-Microverse/assets/107632780/00df5a17-555b-4594-9d88-b68a4c3b3ab8)

Reply:
Added the MIT.md  file and linked the README MIT.MD link properly.